### DISCUSSION TITLE
when i am trying add icon in manifest.json this error is triggers what to do now

### DISCUSSION THREAD

**Discussion by vijaypotnuru** [open-sauced-craftwork/browser-extension#33] [General]
![image](https://user-images.githubusercontent.com/110402074/226188849-059c63ed-21cb-4084-a996-05c8803be9f7.png)

**Comment by nickytonline** [comment]
@vijaypotnuru, the answer is in the error message, `Error: The "fileName" or "name" properties of emitted files must be strings that are neither absolute nor relative paths, received "./logo.png".`

`"./logo.png"` is a relative path.

```diff
  "icons": {
-    "16": "./logo.png"
+    "16": "logo.png"
  }
```

**Reply by vijaypotnuru** [reply]
my problem solved thank you
